core/src/main/java/org/jdbi/v3/core/mapper/EnumMapper.java

```diff
@@ -78,6 +80,16 @@ public E map(ResultSet rs, int columnNumber, StatementContext ctx) throws SQLExc
         private static Object getValueByName(Class<? extends Enum<?>> enumClass, String name) {
             final Enum<?>[] enumConstants = enumClass.getEnumConstants();
             return JdbiOptionals.findFirstPresent(
+                    () -> Arrays.stream(enumConstants).filter(e -> {
+                        final Field field;
+                        try {
+                            field = enumClass.getField(e.name());
+                        } catch (final NoSuchFieldException ex) {
+                            throw new RuntimeException(ex);
```

Comment:
same as before: please use `Unchecked` and consider a private method ref instead of a lambda